packages/compiler/src/core/types.ts

```diff
@@ -2501,18 +2501,18 @@ export interface DecoratorImplementations {
 export interface PackageFlags {
   /**
    * Decorator arg marshalling algorithm. Specify how TypeSpec values are marshalled to decorator arguments.
-   * - `lossless` - New recommended behavior
```

Comment:
Was this the old name for `new`? Haven't actually encountered this :)

Reply:
lossless was actually the original name in the pr but  changed to `new` during review and missed some docs change